### PR TITLE
buffer geocode result

### PR DESCRIPTION
**Pull request by afili**
This sample locates an address using the Locator class and then generates a buffer of 1000 meters around the locator’s result. All features within this buffer are displayed on the map and a message is printed on the left pane.

### CODE REVIEW

File: web-js/buffer-geocoder-result/index.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+              query.geometry = circle.getExtent();
+              //use a fast bounding box query. will only go to the server if bounding box is outside of the visible map
+              featureLayer.queryFeatures(query, selectInBuffer);
+			  
+              return false; //break out of loop after one candidate with score greater  than 80 is found.
+            }
+          });
+          if ( geom !== undefined ) {
+            map.centerAndZoom(geom, 14);
+		  }
```
Comment (jgravois) [inline web-js/buffer-geocoder-result/index.html]: this is nitpicky, but your tabbing/formatting is a little wonky in places.

also, it looks like you are querying using the boundingBox extent of the Circle geometry (as opposed to the circle itself).  i don't think thats a problem, but probably worth mentioning in the readme.

Comment (nixta) [inline web-js/buffer-geocoder-result/index.html]: Is it really that much quicker to do a bounding box vs the circle? Depends on the aim of the sample, but it'd be a lot less code and potentially fewer results across the wire to just query against the circle. If the sample is meant to show polygon contains() on the client, I'd suggest a more complex polygon?

**Comment by jgravois** [comment]
confirmed:
1. code is well formatted
2. the logic is easy to understand
3. it runs as expected.

'squashing' commits isn't mandatory as long as the person confirms that along the way, whether in 1 commit or 10 commits no other files were altered accidently.  that being said, lets give it a try.

it took me a lots of squashing before i memorized the steps, and [this](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request) is still my cheat sheet.

since you applied your commits directly to the `buffer-geocode-result` branch of your own fork, you’d adjust as follows…

```
$ cd into the appropriate folder
$ git fetch upstream
$ git checkout buffer-geocode-result 
$ git rebase -i upstream/master

< choose squash for all of your commits, except the first one >
< Edit the commit message to make sense, and describe all your changes >

$ git push origin buffer-geocode-result -f
```
this will only work if you’ve correctly configured tracking of an upstream repository.
```
$ git remote -v 
>>>needs to return
origin	https://github.com/afili/developer-support.git (fetch)
origin	https://github.com/afili/developer-support.git (push)
upstream	https://github.com/Esri/developer-support.git (fetch)
upstream	https://github.com/Esri/developer-support.git (push)
```

check out [this](https://github.com/Esri/developer-support/pull/5#issuecomment-72518951) thread for more info and let me know if you have any trouble.

**Comment by jgravois** [comment]
i like this sample a lot.  thanks for going to the extra trouble of squashing.

**Comment by afili** [comment]
thanks for helping me with the squash @jgravois ! The trouble was worth it, I like how clean the PR looks now!

**Comment by kurtdaradics** [comment]
weird. i unsubscribed and these e-mails stopped. and now they are magically
reappearing.

**Pull request merged**